src/app/components/FinanceV2Components/PieChart/index.tsx

```diff
@@ -27,7 +29,13 @@ export const PieChart: React.FC<IPieChartProps> = ({
         )}
         secondPercentage={convertPercentageToDegrees(secondPercentage)}
         firstColor={getAssetColor(firstAsset)}
-        secondColor={secondAsset ? getAssetColor(secondAsset) : ''}
+        secondColor={
```

Comment:
Don't use nested ternary expressions, create a method that will return the color and use it here.